dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomImport.scala

```diff
@@ -17,6 +17,7 @@ object LagomImport extends LagomImportCompat {
 
   val lagomJavadslApi = component("lagom-javadsl-api")
   val lagomJavadslClient = component("lagom-javadsl-client")
+  val lagomJavadslIntegrationClient = component("lagom-javadsl-integration-client")
```

Comment:
this was missing, probably because initially not intended to be used in Lagom apps